Ginger/GingerCoreNET/WorkSpaceLib/WorkSpace.cs

```diff
@@ -384,51 +390,63 @@ public bool OpenSolution(string solutionFolder)
                 SolutionUpgrade.ClearPreviousScans();
                 if (SolutionUpgrade.IsGingerUpgradeNeeded(solutionFolder, solutionFiles))
                 {
+                    Console.WriteLine("Error: Current Ginger version");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)